File: sdks/swift/Sources/PuxbaySDK/Models/StockTransfer.swift
import Foundation

public struct StockTransfer: Codable {
    public let id: String
    public let sourceBranch: String
    public let sourceBranchName: String?
    public let destinationBranch: String
    public let destinationBranchName: String?
    public let status: String
    public let items: [[String: String]]?
    public let notes: String?
    public let createdAt: String
    public let updatedAt: String
}

File: sdks/rust/src/models/stock_transfer.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub struct StockTransfer {
    pub id: String,
    pub source_branch: String,
    pub source_branch_name: Option<String>,
    pub destination_branch: String,
    pub destination_branch_name: Option<String>,
    pub status: String,
    pub items: Option<Vec<serde_json::Value>>,
    pub notes: Option<String>,
    pub created_at: String,
    pub updated_at: String,
}
